### PR TITLE
Update CacheResponse.php

### PR DESCRIPTION
**Pull request by PubliAlex**
Only cache pages for anonymous users

### CODE REVIEW

File: classes/middleware/CacheResponse.php [hunk 1]
```diff
@@ -59,6 +60,7 @@ protected function shouldCache(Request $request, $response)
         return $request->isMethod('GET')
             && $request->getQueryString() == null
             && $response->getStatusCode() == 200
+            && BackendAuth::getUser() == null
```
Comment (FlusherDock1) [inline classes/middleware/CacheResponse.php]: Use in this case BackendAuth::check()

**Comment by PubliAlex** [comment]
Updated the PR by following your advice

**Review by FlusherDock1 — CHANGES_REQUESTED** [review]
`BackendAuth::check()` returns boolean value (true or false). Please change comparison from null to false. And i will merge it. Thanks

**Comment by PubliAlex** [comment]
Done :)

**Review by FlusherDock1 — APPROVED** [review]

**Pull request merged**